playbooks/roles/mattermost/templates/autolink.json.j2

```diff
@@ -0,0 +1,18 @@
+# The lines below should be included in the section PluginSettings -> Plugins -> mattermost-autolink
+# in the file config.json.  Since config.json is owned by Mattermost, and JSON does not support any
+# comments, it's difficult to automate this with Ansible.
```

Comment:
But now that it's rendered and manually added to `config.json` already, couldn't we add it to the template anyway, to make it truly automated for future deployments?

I guess the main use of leaving it as a separate config file that has to be manually merged is if we foresee that `MATTERMOST_JIRA_DOMAINS` will change relatively often, forcing us to do something awkward to see the rendered result and change `config.json` manually anyway. If we foresee this variable changing then nevermind this comment.

Reply:
@UmanShahzad We add a new project to our own Jira instance every few months, and this requires an update to `MATTERMOST_JIRA_DOMAINS` everytime.  I chose the design you reviewed to make these updates relatively painless – we just need to copy the few lines from the one file to the other.

However, based on your comment, I slightly modified the solution so it builds the full configuration on fresh installs – you are right that we should do that given that it's so simple now.  I had to turn the fake comment into a Jinja2 comment to prevent it from showing up in the output, since JSON files can't have comments.

Reply:
I tested the new version by moving `config.json` on the server out of the way, running the playbook to generate a new version and verifying that both files are essentially identical, which they are.